api/desecapi/models/tokens.py

```diff
@@ -65,7 +65,7 @@ class Meta:
             # Ensure that a default policy is defined when auto_policy=true
             pgtrigger.Trigger(
                 name="token_auto_policy",
-                operation=pgtrigger.Update | pgtrigger.Insert,
+                operation=pgtrigger.Insert | pgtrigger.UpdateOf("auto_policy"),
```

Comment:
This one ensures that `auto_policy` checks don't run when it's not being changed (e.g., when only changing `last_used` during authentication)